code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/impl/ApiAuthenticator.java

```diff
@@ -53,15 +56,15 @@ public void handle(ApiRequest request) throws IOException {
 
         Policy policy = getPolicy(account, request);
         if (policy == null) {
-            log.error("Failed to find policy for [{}]", account.getId());
-            throw new ClientVisibleException(ResponseCodes.UNAUTHORIZED);
+            log.error("Failed to find policy for [{}]", account.getAccount().getId());
```

Comment:
To prevent this from looking weird, I wonder should we rename the variable 'account' to 'accountAccess'?
Otherwise, account.getAccount().getId() looks weird.